contrib/cirrus/lib.sh

```diff
@@ -244,6 +245,28 @@ use_netavark() {
     # N/B: The CNI packages are still installed and available. This is
     # on purpose, since CI needs to verify the selection mechanisms are
     # functional when both are available.
+
+    # See ./contrib/cirrus/CIModes.md.
+    # Vars defined by cirrus-ci
+    # shellcheck disable=SC2154
+    if [[ "$CIRRUS_CHANGE_TITLE" =~ CI:[AN]V[AN]V= ]]; then
+        # shellcheck disable=SC2154
+        if [[ "$CIRRUS_PR_DRAFT" != "true" ]]; then
+            die "Magic 'CI:NVAV=*' string can only be used on DRAFT PRs"
+        fi
+
+        magickind=$(sed -r -e 's~(.*CI:[AN]V[AN]V=)(\w+)(.*)~\2~' <<<"$CIRRUS_CHANGE_TITLE")
+        if [[ "$magickind" == "update" ]]; then
+            warn "Updating released netavark/aardvark RPM packages from updates repo."
+            showrun dnf update -y netavark aardvark-dns
```

Comment:
One could move lines 261 and 265, which are identical, down to just after line 268. If one were neurotic about removing unnecessary duplication, and about being polite to future maintainers who will otherwise spend time trying to understand what's the difference between those two `dnf` commands...